scripts/release/autorelease-step-2.sh

```diff
@@ -0,0 +1,76 @@
+# Finish the release.
+# - Download the build artifacts
+# - Create a github release for the tag
+#   - Attach release notes
+#   - Attach the build artifacts
+# - Create a pull request from the autorelease branch into main
+# - Create a pypi release
+#   - Attach the build artifacts
+set -e  # Exit the script immediately if any subshell has a nonzero exit code.
+
+VERSION=$1
+GH_RUN_ID=$2
+GITHUB_REPO="natcap/invest"
+PYPI_REPO="testpypi"
```

Comment:
I am leaving this set to testpypi for now so we don't accidentally create a permanent release on pypi. To run the script for real, just change this to "pypi"

Reply:
I like the idea of having this be "testpypi", but then after the script completes maybe we can have instructions saying something like:

```sh
echo "Release has been created using testpypi, to release officially run:"
echo "twine upload -r pypi dist/natcap.invest.*"
```

Reply:
Sounds good